src/Utils/Rating/index.js

```diff
@@ -0,0 +1,8 @@
+import { Rate } from 'antd'
+const Rating = (rating) => {
+  return (
```

Comment:
use jsx format extesion (.jsx)

Reply:
thankyou for your fix 2313f342525894ae95ccbcbbe4d0766e8bf1e34a